lib/__tests__/standalone.test.js

```diff
@@ -114,10 +115,27 @@ it("standalone without input css and file(s) should throw error", () => {
   );
 });
 
-it("standalone with non-existent-file quietly exits", () => {
+it("standalone with non-existent-file throws an error", () => {
+  const files = `${fixturesPath}/non-existent-file.css`;
+  const expectedError = new NoFilesFoundError(files);
+
   return standalone({
-    files: `${fixturesPath}/non-existent-file.css`,
+    files,
     config: configBlockNoEmpty
+  })
+    .then(() => {
+      throw new Error("should not have succeeded");
```

Comment:
I'm just curious why to throw an error here? Isn't stylelint should throw an error and `catch` will catch it?

Reply:
Hi, I removed this, it was left over from testing